### PR TITLE
Fix ranking video gallery shift

### LINKED ISSUE #1575: Layout shift in numbered carousel
Once the loading is finished, you can see the numbers moving down:

https://user-images.githubusercontent.com/12646744/140962104-2d04a777-0832-4dfd-909d-64d79f5135bd.mov

### PR DESCRIPTION
**Pull request by rafalpawlow**
Fix #1575

### CODE REVIEW

**Review by eldiegod — APPROVED** [review]
Does the trick

File: src/shared/components/VideoTileBase/VideoTileBase.styles.tsx [hunk 1]
```diff
@@ -85,6 +85,15 @@ export const Container = styled.article<LoadingProps>`
...
   flex-direction: column;
+  min-height: 250px;
+
+  ${media.sm} {
+    min-height: 260px;
+  }
+
+  ${media.xxl} {
+    min-height: 270px;
+  }
```
Comment (kdembler) [inline src/shared/components/VideoTileBase/VideoTileBase.styles.tsx]: Is the video tile height really the problem here? I would expect that this is more of an layout/alignment issue with the big number.

**Review by kdembler — CHANGES_REQUESTED** [review]
Problem still persists on some screen sizes:

https://user-images.githubusercontent.com/12646744/141782979-da4437ac-e0e1-4865-9145-dae0387886d4.mov

File: src/shared/components/VideoTileBase/VideoTileBase.styles.tsx (hunk 1 above)
Comment (rafalpawlow) [reply src/shared/components/VideoTileBase/VideoTileBase.styles.tsx]: Unfortunately it uses `align-items: center;`, problem appears when loaded video has more lines than skeleton. I will try figure out some other way to position it.

**Review by kdembler — APPROVED** [review]

**Pull request merged**